torch/_prims_common/__init__.py

```diff
@@ -265,12 +265,14 @@ def is_contiguous(a: TensorLikeType, false_if_dde=False) -> bool:
     from torch.fx.experimental.symbolic_shapes import (
         guard_or_false,
         guard_or_true,
-        guard_size_oblivious,
         is_nested_int,
     )
 
-    maybe_guard_or_false = guard_or_false if false_if_dde else guard_size_oblivious
-    maybe_guard_or_true = guard_or_true if false_if_dde else guard_size_oblivious
+    def eval_eager(x):
```

Comment:
shall we just do torch.ops.aten.sym_is_contiguous.default().guard_or_false here
make debugging harder, but make impl simpler.

Reply:
Does it make debugging harder? It early exits the false case and turns all the following guard_or_false/true below to just a simple "if", doesn't it?
(Same answer for is_channels_last_contiguous_2d)